### PR TITLE
Various fixes 2

### PR DESCRIPTION
**Pull request by marceloyuela**
Only one cmdlet needed to have the positional parameter of InstanceObject changed. Everything else was fine.

### CODE REVIEW

File: Google.PowerShell/Sql/GcSqlSslCmdlets.cs [hunk 1]
```diff
@@ -149,7 +149,7 @@ protected override void ProcessRecord()
     ///   Adds the SSL Certificate called "myCert" to the instance stored in $myInstance.
     ///   </para>
     ///   <para><code>
-    ///     PS C:\> Add-GcSqlSslCert $myInstance "myCert"
+    ///     PS C:\> Add-GcSqlSslCert "myCert" -InstanceObject $myInstance
```
Comment (ILMTitan) [inline Google.PowerShell/Sql/GcSqlSslCmdlets.cs]: Why not give the pipelining example?

**Comment by ILMTitan** [comment]
Just one comment on your example, but LGTM.

**Comment by marceloyuela** [comment]
Fixed that example.

**Comment by ILMTitan** [comment]
LGTM.

**Pull request merged**